### ISSUE TITLE
Allow initial entity type filter to be set in DefaultCatalogPage

### ISSUE THREAD

**Issue by zhammer** [backstage/backstage#10313]
## Feature Suggestion

Our org (SeatGeek) is centralizing on the `Kind: System` entity as our "core" entity. We'd like the initial entity type filter to be `Kind: System`. This isn't settable in the `DefaultCatalogPage` currently. Of course it may be the intention to push people towards creating a [custom catalog page](https://backstage.io/docs/features/software-catalog/catalog-customization).. but given...
- there is [already a prop for initial user filter](https://github.com/backstage/backstage/blob/9929e2402c8347fdd6a38ae9d6bb6424b2e129bf/plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.tsx#L48) (though perhaps too generically named)
- EntityTypePicker has an [initialFilter prop](https://github.com/backstage/backstage/blob/9929e2402c8347fdd6a38ae9d6bb6424b2e129bf/plugins/catalog-react/src/components/EntityTypePicker/EntityTypePicker.tsx#L31)
- Creating a custom catalog page for such a small customization leads to potential migration pain & backstage drift

... I think passing this prop through would be great!

## Possible Implementation

Add a new prop to defaultcatalogpage, `initiallySelectedEntityTypeFilter?: EntityTypePickerKind`, pass that through to `EntityTypePicker`

## Context

customization of our catalog without the cost of needing to maintain our own catalog page implementation

**Comment by freben** [comment]
Oh sorry - this is what you want, right? https://github.com/backstage/backstage/pull/10291

**Comment by zhammer** [comment]
ah, perfect!

**Comment by zhammer** [comment]
will this issue be linked when the change ships? dont think it’s been published yet and am curious if i need to keep checking the changelog or if i’ll get a notification for a comment on this thread!

**Comment by freben** [comment]
No, it won't be seen in here. Once it's in master, it'll be in any following release. The nearest `next` (weekly) release is on Tuesday, if you follow those pre-release rather than the mainline
